### ISSUE TITLE
Synchronized Judging Stab

### ISSUE THREAD

**Issue by prphntm63** [CIA-Homebrew/BJCP-Scoresheet#86]
Propose / whiteboard methodology to ensure flights are judged simultaneously. 

Ideas:
* Flight time limit?
* Photo verification?

**Comment by PH-Lazurous** [comment]
What's the driving factor behind this? Just to make sure they get done at the same time across judges or is there a specific requirement for them to be done synchronously?

Ultimately I would this this needs to be 'managed' via a head judge, queue your judges, assign the flights and the 'activate' the scoresheets so they can begin work. Then your time limit would be the simplest. Photo verification of what? empty cup? 

Just trying to wrap my head around this a bit.

**Comment by prphntm63** [comment]
Correct, some way to make sure that two flights are judged synchronously. This "issue" would involve talking to comp admins and getting their feedback about the best way to do this, then creating a new issue with a plan going forward for implementation.
